apps/web/pages/v2/apps/[slug]/setup.tsx

```diff
@@ -12,11 +12,7 @@ export default function SetupInformation(props: InferGetStaticPropsType<typeof g
   const { status } = useSession();
 
   if (status === "loading") {
-    return (
-      <div className="absolute z-50 flex h-screen w-full items-center bg-gray-200">
-        <Loader />
```

Comment:
NIT: import Loader from "@calcom/ui/Loader";

still being imported

Reply:
Good catch! I removed the import 👍